interpreter/cling/lib/Interpreter/DeclUnloader.cpp

```diff
@@ -602,7 +602,8 @@ namespace cling {
     // llvm::Module cannot contain:
     // * variables and parameters with dependent context;
     // * mangled names for parameters;
-    if (!isa<ParmVarDecl>(VD) && !VD->getDeclContext()->isDependentContext()) {
+    if (!isa<ParmVarDecl>(VD) && !VD->getDeclContext()->isDependentContext() &&
+        VD->getIdentifier()) {
```

Comment:
I imagine that the anonymous variables are given some internal name. Can we undo that instead of not undoing anything?